packages/frontend/src/components/Intercom.tsx

```diff
@@ -1,11 +1,31 @@
 import { LightdashMode } from 'common';
 import React, { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
 import { useIntercom } from 'react-use-intercom';
 import { useApp } from '../providers/AppProvider';
 
+const LOCATIONS_WITH_INTERCOM = ['/welcome', '/register', '/login', '/invite'];
```

Comment:
> Connect a project page (during onboarding, as this is where users need the most help)

are we not missing /createProject ?

Reply:
It would show in front of the new steps toast at the bottom. At this point the user is inside the app and have access to the support via top navbar. Talked with @PriPatel and we decided to not show the launcher in that page.

Reply:
cc @hamzahc1